sympy/solvers/pde.py

```diff
@@ -897,11 +897,11 @@ def pde_separate_add(eq, fun, sep):
     """
     Helper function for searching additive separable solutions.
 
-    Consider an equation of two independent variables x, y and a dependent
-    variable w, we look for the product of two functions depending on different
+    Consider an equation of two independent variables x and t and a dependent
```

Comment:
Is it possible though that the summands can depend on more than one of the independent variables?  In the text of the current help-string (but not its code example) in master, the dependent variable w(x, y, z) is split into two, with the second depending on y and z.  (Mathematically this is certainly possible; I haven't checked whether the implementation supports this.)